### ISSUE TITLE
Kernel dies

### ISSUE THREAD

**Issue by vnt1537** [anasmorahhib/3D-CNN-Gesture-recognition#5]
# Normalisation: training
print('old mean', training_data.mean())
scaler = StandardScaler()
scaled_images  = scaler.fit_transform(training_data.reshape(-1, 15*64*64))
print('new mean', scaled_images.mean())
scaled_images  = scaled_images.reshape(-1, 15, 64, 64, 1)
print(scaled_images.shape)    

When executing this command my kernel dies. Can you give any solution?

**Comment by anasmorahhib** [comment]
How much RAM do you have?
try to run the script on the cloud.
